### ISSUE TITLE
so many boxes drawed

### ISSUE THREAD

**Issue by ghost** [thtrieu/darktf#1191]
Hi
I'm learning new data without pretrained data(like '.weight' file).
However, I don't know if the learning is going well.

I tried several times before, but it didn't work, so I started training data anew.

threshold 0.5 -> no box
threshold 0.003.... 0.0003 -> so many boxes

when draw the box threshold at 0.5, Will the box be drawn when it is over 10,000 times?

Any help?

_I am learning the license plate, and the class is 11 / Image is 29.
step 500 
loss 20.xxx..._

**Comment by mahmoudalbardan** [comment]
Hello, did you find what was causing your issue?
